openadapt/record.py

```diff
@@ -606,11 +685,37 @@ def read_keyboard_events(
     terminate_event: multiprocessing.Event,
     recording_timestamp: float,
 ) -> None:
+    """Reads keyboard events and adds them to the event queue.
+
+    Args:
+        event_q (queue.Queue): The event queue to add the keyboard events to.
+        terminate_event (multiprocessing.Event): The event to signal termination
+          of event reading.
+        recording_timestamp (float): The timestamp of the recording.
+
+    Returns:
+        None
+    """
     # create list of indices for sequence detection
     # one index for each stop sequence in STOP_SEQUENCES
     stop_sequence_indices = [0 for _ in STOP_SEQUENCES]
 
-    def on_press(event_q, key, injected):
+    def on_press(
+        event_q: queue.Queue,
+        key: keyboard.KeyboardEvent,
```

Comment:
`key: Union[keyboard.Key, keyboard.KeyCode]`

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/87188ec08861da59a980d007fa64efda02f222e8